templates/applicants.phtml

```diff
@@ -42,6 +43,7 @@
                     <td><a href="/admin/applicants/<?= $applicant->getId() ?>"><?= $applicant->getName(); ?></a></td>
                     <td><?php echo $applicant->getEmail(); ?></td>
                     <td><?php echo $applicant->getFormattedApplicationDate(); ?></td>
+                    <td><a href="/admin/applicant/edit/<?= $applicant->getId() ?>" class="edit-button">EDIT</a></td>
```

Comment:
It's really easy to make the button consistent with other buttons because there is already a CSS class...

```suggestion
                    <td><a href="/admin/applicant/edit/<?= $applicant->getId() ?>" class="btn">EDIT</a></td>
```

Don't reinvent the wheel.